components/modules/Header/Header.tsx

```diff
@@ -1,112 +1,126 @@
-  <header className={styles.header}>
-    <div className={`container ${styles.content}`}>
-      <div className={styles.top}>
-        <div className={styles.logo}>
-          <LogoPNG />
-          <div className={styles.gallery}>
-            <p>Галерея молодых и</p>
-            <p>малоизвестных художников</p>
-            <p>Новое пространство</p>
+  const handleOpenMenu = () => {
+    addOverflowHiddenToBody()
+    dispatch(openBurgerMenu())
+  }
+  return (
+    <header className={styles.header}>
+      <div className={`container ${styles.content}`}>
+        <div className={styles.top}>
+          <div className={styles.logo}>
+            <LogoPNG />
+            <div className={styles.gallery}>
+              <p>Галерея молодых и</p>
+              <p>малоизвестных художников</p>
+              <p>Новое пространство</p>
+            </div>
           </div>
-        </div>
-        <div className={styles.social}>
-          <a
-            className={styles.tg}
-            href='https://t.me/newartspace'
-            target='_blank'
-          />
-          <a
-            className={styles.vk}
-            href='https://vk.com/public207408538'
-            target='_blank'
-          />
-        </div>
-        <div className={styles.contacts}>
-          <p>
-            <a className={styles.email} href='mailto:rodoc461@aersm.com'>
-              9326215@mail.ru
-            </a>
-          </p>
-          <p>
-            <a className={styles.phone} href='tel:89219326215'>
-              +7 (921) 932-62-15
-            </a>
-          </p>
-        </div>
-        <div className={styles.work_time}>
-          <p>Пн – Пт с 13:00 до 19:00</p>
-          <p>Сб – Вс по предварительной договоренности</p>
-        </div>
-        <div className={styles.links}>
-          <div className={styles.cart}>
-            <Link href='/'>
-              <CartSVG />
-            </Link>
+          <div className={styles.social}>
+            <a
+              className={styles.tg}
+              href='https://t.me/newartspace'
+              target='_blank'
+            />
+            <a
+              className={styles.vk}
+              href='https://vk.com/public207408538'
+              target='_blank'
+            />
+          </div>
+          <div className={styles.contacts}>
+            <p>
+              <a className={styles.email} href='mailto:rodoc461@aersm.com'>
+                9326215@mail.ru
+              </a>
+            </p>
+            <p>
+              <a className={styles.phone} href='tel:89219326215'>
+                +7 (921) 932-62-15
+              </a>
+            </p>
+          </div>
+          <div className={styles.work_time}>
+            <p>Пн – Пт с 13:00 до 19:00</p>
+            <p>Сб – Вс по предварительной договоренности</p>
           </div>
-          <div className={styles.user}>
-            <Link href='/'>
-              <UserSVG />
-            </Link>
+          <div className={styles.links}>
```

Comment:
Unnecessary `<div>` wrapper for `<Link>`. You can just use `<Link>` with ::before styles for icons without svg